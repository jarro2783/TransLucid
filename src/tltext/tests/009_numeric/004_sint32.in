%%
sint32"0";;
sint32"-2147483648";;
sint32"2147483647";;

sint32"-500" + sint32"1000";;
sint32"-500" - sint32"1000";;
sint32"500" * sint32"1000";;
sint32"-500" * sint32"1000";;
sint32"50" / sint32"10";;
sint32"50" / sint32"9";;
sint32"50" % sint32"8";;

sint32"0" < sint32"56";;
sint32"0" < sint32"0";;
sint32"0" <= sint32"56";;
sint32"0" <= sint32"0";;
sint32"0" > sint32"56";;
sint32"0" >= sint32"56";;
sint32"56" >= sint32"56";;
sint32"0" == sint32"56";;
sint32"56" == sint32"56";;
sint32"0" != sint32"56";;
sint32"56" != sint32"56";;
