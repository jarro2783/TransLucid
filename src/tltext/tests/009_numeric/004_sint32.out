sint32"0"
sint32"-2147483648"
sint32"2147483647"
sint32"500"
sint32"-1500"
sint32"500000"
sint32"-500000"
sint32"5"
sint32"5"
sint32"2"
true
false
true
true
false
false
true
false
true
true
false
